bot/exts/help_channels/_cog.py

```diff
@@ -279,6 +286,14 @@ async def init_cog(self) -> None:
         # This may confuse users. So would potentially long delays for the cog to become ready.
         self.close_command.enabled = True
 
+        # Acquiring the dynamic message ID, if it exists within the cache.
+        log.trace("Attempting to fetch How-to-get-help dynamic message ID.")
+        self.dynamic_message = await _caches.dynamic_message.get("message_id")
+
+        # Getting channels that need to be included in the dynamic message.
+        await self.update_available_help_channels()
+        log.trace("Dynamic available help message updated.")
```

Comment:
Perhaps this should be moved down below the `init_available` call. Makes more sense to wait for the channels to be synced before we edit the message.

Reply:
Will do, thanks!